Shared/Sources/SectionItem.swift

```diff
@@ -0,0 +1,10 @@
+// Copyright © 2020 Itty Bitty Apps. All rights reserved.
+
+import Foundation
+
+protocol SectionItem: Decodable {
+  associatedtype item
```

Comment:
Same as the earlier comment: this type alias should probably start with an uppercase letter.